src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -164,6 +164,8 @@ public static void BuildDefaultConfigurationFile(StringBuilder builder, Network
             builder.AppendLine($"#Enable bandwidth saving setting to send and received confirmed blocks only. Defaults to { (DefaultBlocksOnly ? 1 : 0) }.");
             builder.AppendLine($"#blocksonly={ (DefaultBlocksOnly ? 1 : 0) }");
             builder.AppendLine($"#bantime=<number>");
+            builder.AppendLine($"#Disallow connection to peers in same IP range. Default is true for remote hosts.");
+            builder.AppendLine($"#iprangefiltering=<true or false>");
```

Comment:
is it really true/false or is it 1/0?

Reply:
Correcting this to to 1,0 to be consistent with other settings.  Also changed PrintHelp for consistency.